lib/matplotlib/tests/test_colors.py

```diff
@@ -156,6 +156,193 @@ def test_LogNorm():
     assert_array_equal(ln([1, 6]), [0, 1.0])
 
 
+def test_FuncNorm():
```

Comment:
I guess the line of the assert is always shown, but in any case, could you show me an example of parameterized, please?